microservices/nris_api/backend/app/extensions.py

```diff
@@ -0,0 +1,28 @@
+import cx_Oracle
+from flask_caching import Cache
+from flask_jwt_oidc import JwtManager
+from flask_sqlalchemy import SQLAlchemy
+from flask_apscheduler import APScheduler
+from flask_migrate import Migrate, MigrateCommand
+from flask import current_app
+from elasticapm.contrib.flask import ElasticAPM
+
+from .config import Config
+from .helper import Api
+
+apm = ElasticAPM()
```

Comment:
Is the APM complete?

Reply:
local doesn't hook up to APM, depends on if the elastic env variables are being set in the deployment config properly, which i have not validated. Should something be set/changed to differentiate this traffic from the core_api traffic?